Misc/NEWS.d/next/Library/2023-11-29-07-36-02.gh-issue-83055.Xd4kZv.rst

```diff
@@ -0,0 +1 @@
+Fix a bug on asyncio.Queue
```

Comment:
We're usually a bit more specific in news entries. Also, it's ReST, so needs markup like we use in Doc (.rst files). And punctuation.

Reply:
Update done.